xml/System.Diagnostics/TagList.xml

```diff
@@ -426,7 +426,8 @@
       </ReturnValue>
       <Docs>
         <summary>Gets a value indicating whether the <see cref="T:System.Diagnostics.TagList" /> is read-only. This property will always return <see langword="false" />.</summary>
-        <value>To be added.</value>
+        <value>
+          <see langword="true" /> if the <see cref="T:System.Collections.Generic.ICollection`1" /> is read-only; otherwise, <see langword="false" />.</value>
```

Comment:
This contradicts with the summary, which says it always returns false.

Reply:
@tarekgh

Reply:
@gewarren is correct. We always return false here.

Reply:
```suggestion
          Always returns <see langword="false" />.</value>
```